### ISSUE TITLE
I have no idea what I am doing.

### ISSUE THREAD

**Issue by LoafPickleWW** [algorandfoundation/xGov#5]
As a person who does not use github, I have tried following directions and have no clue what I am doing.

Why github? Why make this confusing for people that aren't devs? Why not put this on the xgov page and have a simple web format with wallet connect? 

Come on all, we need to make this chain accessible to normal people who don't use code repositories.

**Comment by richflairstv** [comment]
Completely agree ☝️

**Comment by dkpatton** [comment]
I disagree. Though not very easy to use, it is portable, transparent, and secure. I can't think of an alternative that I would trust.

**Comment by johnalanwoods** [comment]
What alternatives would the community prefer?

**Comment by LoafPickleWW** [comment]
> What alternatives would the community prefer?

The xGov page on the website should have a wallet connect feature (w/ auth) and when you log in you can: commit to xgov, add proposals via a form, review submitted proposals, vote on proposals, and receive your deposit. 

Everything should be in one spot. That would be the best UX. 

It was lame having to follow a github guide and go through a maze of links just to propose something. I went through the process of proposing something and it was not fun. If I wasn't trying to help with feedback, I would have never done it. I saw this as a big obstacle to get new people here building. There is this sinking perception that the dev community from AF/INC don't care about the end users (I know you all do and it isn't true), but it is things like this that help make that mindset fester.

**Comment by joe-p** [comment]
I think we all agree a all-in-one dapp would be better, but that would take a lot of time and resources to build out. We need a tool that has clear versioning and commenting. There are other features on Github that are nice like line-specific comments, but to me proposal/comment history is the most important. 

So, the question is what tools can we leverage to do this? The first obviously comes to mind is Google forms + docs, but the problem is that google docs don't have a great versioning scheme. You can view file history, but there's no way to link to specific versions and comments can be hard to follow between versions.

**Comment by LoafPickleWW** [comment]
> I think we all agree a all-in-one dapp would be better, but that would take a lot of time and resources to build out. We need a tool that has clear versioning and commenting. There are other features on Github that are nice like line-specific comments, but to me proposal/comment history is the most important.
> 
> So, the question is what tools can we leverage to do this? The first obviously comes to mind is Google forms + docs, but the problem is that google docs don't have a great versioning scheme. You can view file history, but there's no way to link to specific versions and comments can be hard to follow between versions.

Are there ways to read/write to github with a user friendly front end?

**Comment by bgsnyder84** [comment]
Agree, I don't know what I'm going here, and not sure how this process keeps people who didn't commit to xgov from participating, although that may be because I don't know what I'm doing here.

**Comment by joe-p** [comment]
> Are there ways to read/write to github with a user friendly front end?

None that I'm aware of. It would be possible though with the GitHub API

**Comment by jbenjamins81** [comment]
Is it mandatory to build a proposal in GitHub to keep the XGov status? Is voting on other proposals enough?

**Comment by Coder-aadarsh** [comment]
While some argue that GitHub offers portability, transparency, and security, it's evident that the complexities and unfamiliarity with the platform deter potential contributors.
While Google Docs in my opinion falls short due to its lack of robust version control. However, seeking ways to integrate GitHub's features through a more user-friendly front end, potentially via the GitHub API, could be explored

**Comment by nonfungibleab** [comment]
Thanks @aadarsh-nagrath. We are certainly exploring integrations as we continue to develop the program.
